model/registry/test.yaml

```
@@ -0,0 +1,85 @@
+groups:
+  - id: registry.test.suite
```

```suggestion
  - id: registry.test
```

I think this should fix the title in the file

---

Yup, it did, thanks!